Jenkinsfile

```diff
@@ -88,3 +102,21 @@ def AMD64BuildGPU() {
     }
   }
 }
+
+def CloudInstallAndTest(cloudTarget) {
+  def nodeReq = "ubuntu && amd64 && ${cloudTarget}"
+  node(nodeReq) {
+    echo "Installing DLR package for ${cloudTarget} target"
+    withAWS(credentials:'Neo-AI-CI-Fleet') {
+      files = s3FindFiles(bucket: 'neo-ai-dlr-jenkins-artifacts', glob: "${env.JOB_NAME}/${env.BUILD_ID}/artifacts/dlr-*-manylinux1_x86_64.whl")
```

Comment:
How are these files in `neo-ai-dlr-jenkins-artifacts` generated?

Reply:
In the Build stage

Reply:
Got it. Thank you!